tests/ImageSharp.Tests/Processing/Processors/Convolution/BokehBlurTest.cs

```diff
@@ -138,21 +138,10 @@ public void Serialize(IXunitSerializationInfo info)
         public void BokehBlurFilterProcessor<TPixel>(TestImageProvider<TPixel> provider, BokehBlurInfo value)
             where TPixel : unmanaged, IPixel<TPixel>
         {
-            static void RunTest(string providerDump, string infoDump)
-            {
-                TestImageProvider<TPixel> provider =
-                    BasicSerializer.Deserialize<TestImageProvider<TPixel>>(providerDump);
-                BokehBlurInfo value = BasicSerializer.Deserialize<BokehBlurInfo>(infoDump);
-
-                provider.RunValidatingProcessorTest(
-                    x => x.BokehBlur(value.Radius, value.Components, value.Gamma),
-                    testOutputDetails: value.ToString(),
-                    appendPixelTypeToFileName: false);
-            }
-
-            RemoteExecutor
-                .Invoke(RunTest, BasicSerializer.Serialize(provider), BasicSerializer.Serialize(value))
-                .Dispose();
+            provider.RunValidatingProcessorTest(
```

Comment:
This was remote executed to reduce memory pressure, especially on x86. `BokehBlur` is very memory intensive.

Reply:
They seem to be ok now. As I recall it uses less memory than it did originally. Haven't seen any failures in the last few test runs due to this.

Reply:
Definitely rock solid. Multiple test runs with zero issues.

Reply:
Ok for now then, but if we'll see OOM-s after introducing new 32bit targets, this is a good place to get back.

Note that we were getting OOM-s not because of a specific test but rather because of reaching a critical mass of memory intentse-tests. I also pushed some Jpeg tests out-process together with the Bokeh ones back in the past. The price is  execution time OFC. Testing image-processing is hard :)

Reply:
Preach!